src/openmp-validation-1-fixes.patch

```diff
@@ -42,5 +49,56 @@ diff -u OpenMP3.1_Validation/omp_my_sleep.h OpenMP3.1_Validation.new/omp_my_slee
  #include <sys/time.h>
  #include <time.h>
  #include <errno.h>
-Only in OpenMP3.1_Validation.new: ompts.log
-Only in OpenMP3.1_Validation.new: results.txt
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tony Theodore <tonyt@logyst.com>
+Date: Sun, 23 Feb 2014 05:22:26 +1100
+Subject: [PATCH] add Batchfile with test to run on host
+
+
+diff --git a/all-tests-openmp-validation.bat b/all-tests-openmp-validation.bat
```

Comment:
Why should it work incorrectly? We already have many DOS line endings patches created with patch-tool-mxe. File `openmp-validation-2-test-crlf.patch` had mixed line endings: DOS for Copyright header and diff content and Unix for diff header.

Could you explain the problem with DOS line endings and how .gitattributes approaches it, please?